docs/docs/faq.md

```diff
@@ -42,8 +42,8 @@ So, in your `pyproject.toml` file, add this section if it does not already exist
 
 ```toml
 [build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=1.0.0"]
```

Comment:
Is it possible to have `core` somehow namespaced under `poetry` so that we will instead have `requires = ["poetry.core>=1.0.0"]`? Using a dash in the package name can be quite confusing, for example, when do we use the dash and the underscore (in `docs/pyproject.md`,  it requires `poetry_core`).

Reply:
Both of these are equivalent to PyPI - so “_-” won't matter to PyPI. You can use poetry_core instead of poetry-core and they'll refer to the same thing in the dependency specification.

That's the interoperability standard (PEP 508/440) and I expect poetry follows it. :)

Reply:
Ah okay, good to know, thanks!

Reply:
I would have preferred to use a namespace package but, as far as I know, it's not possible since poetry is already its own package.

It would require to no longer have the `poetry` package but multiple packages (like `poetry-core` and `poetry-cli`) so I don't think it's worth it for now.

However, if someone tells me there is a way to do it without big changes I'd be happy to change it.

Reply:
In the end, I managed to make the `poetry` namespace work, so `poetry-core` will provide the `poetry.core` package.